### PR TITLE
Fixes #190 panel title for multiple components

### LINKED ISSUE #190: Multiple components
Options for showing the name of multiple components on the components list:
- 1 The component name itself
  screenshot 2016-06-29 01 52 54
- 2 COMPONENT_NAME (_InstanceID_)
  screenshot 2016-06-29 01 57 11
- 3 INSTANCE_ID (_ComponentName_)
  screenshot 2016-06-29 01 56 26
- 4 COMPONENT_NAME INSTANCEID
  screenshot 2016-06-29 03 16 02

### PR DESCRIPTION
**Pull request by fernandojsg**
Following the comments from @dmarcos and @ngokevin I've changed the style of the multiple components panel title:
screenshot 2016-07-11 22 58 29

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -22,12 +22,13 @@
         <img id="floorimg" src="assets/textures/grasslight-big.jpg">
         <img id="floorimg2" src="assets/textures/grasslight-big.jpg">
       </a-assets>
-<!--      <a-entity id="soundcube" position="0 0 -10" rotation="45 30 0"
-          sound__1="autoplay: true; src: assets/sounds/321103__nsstudios__blip1.wav;"
-          sound__2="autoplay: true; src: assets/sounds/Laser-SoundBible.com-602495617.com-602495617.ogg;"
+      <a-entity id="soundcube" position="0 0 -10" rotation="45 30 0"
```
Comment (ngokevin) [inline example/index.html]: This won't play when we load on GitHub pages, right?

File: src/components/attributes/Component.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ export default class Component extends React.Component {
     return (
       <Collapsible>
         <div className='collapsible-header'>
-          <span>{componentName} <span className='subcomponent'>{subComponentName}</span></span>
+          <span>{subComponentName || componentName}</span>
```
Comment (ngokevin) [inline src/components/attributes/Component.js]: Needs a title to see the full name on hover.

Comment (fernandojsg) [inline src/components/attributes/Component.js]: Done

File: example/index.html (hunk 1 above)
Comment (fernandojsg) [inline example/index.html]: What you mean, that it won't play it automatically? Right now it doesn't have `autoplay: true`  or any event assigned so it won't play at all. Still I'll remove it from the example, I forgot to delete it before pushing

**Pull request merged**